tests/e2e/env/utils/update-ready-page.js

```diff
@@ -25,11 +25,11 @@ const updateReadyPageStatus = async ( status ) => {
 		statusFilter = 'draft';
 	}
 
-	const getPostsResponse = await client.get(`${wpPagesEndpoint}?search=ready&status=${statusFilter}`);
-	if ( getPostsResponse.data && getPostsResponse.data.length > 0) {
+	const getPostsResponse = await client.get(`${wpPagesEndpoint}?search=ready&status=${statusFilter}` );
```

Comment:
Minor changes here to add spacing to the front:

```suggestion
	const getPostsResponse = await client.get( `${wpPagesEndpoint}?search=ready&status=${statusFilter}` );
```